### PR TITLE
chore: remove freenode (community safety concerns)

### PR DESCRIPTION
**Pull request by bnb**
removes freenode for community safety concerns, including the platform's leadership [calling a project member a slur](https://github.com/nodejs/TSC/issues/1031#issuecomment-860812817).

I don't expect immediate merge on this, but wanted to go ahead and put it up (ref: https://github.com/nodejs/TSC/issues/1031)

### CODE REVIEW

File: locale/en/get-involved/index.md [hunk 1]
```diff
@@ -9,7 +9,6 @@ layout: contribute.hbs
 
 * The [GitHub issues list](https://github.com/nodejs/node/issues) is the place for discussion of Node.js core features.
 * For real-time chat about Node.js development use one of the platforms below
-  * For IRC, go to `irc.freenode.net` in the `#node.js` channel with an [IRC client](https://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients) or connect in your web browser to the channel using [freenode's WebChat](https://webchat.freenode.net/#node.js).
```
Comment (ljharb) [inline locale/en/get-involved/index.md:12]: we could replace it with libera.chat in the meantime - it's safely owned and moderated.
```suggestion
  * For IRC, go to `irc.libera.chat` in the `#node.js` channel with an [IRC client](https://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients) or connect in your web browser to the channel using [a web client](https://kiwiirc.com/nextclient/).
```

**Review by mcollina — APPROVED** [review]
lgtm asap

**Review by joesepi — APPROVED** [review]

**Comment by joesepi** [comment]
This will need to happen in all of the locales: https://github.com/nodejs/nodejs.org/search?q=freenode

File: locale/en/get-involved/index.md (hunk 1 above)
Comment (bnb) [reply locale/en/get-involved/index.md:12]: Are we actually moving to libera officially?

Comment (ljharb) [reply locale/en/get-involved/index.md:12]: I think we should; either way, the channel exists and is moderated by the same folks that ran the freenode one.

**Review by ghost — APPROVED** [review]

**Pull request merged**